src/application/wizard/service.ts

```diff
@@ -52,10 +52,17 @@ export class WizardService {
         return submission;
     }
 
+    async deleteManuscriptFile(fileId: FileId, submissionId: SubmissionId, user: User): Promise<boolean> {
```

Comment:
There are no checks using the permission service - see "saveManuscriptFile"

Reply:
Done and pushed